### ISSUE TITLE
REST API: Improve response if a test run fails

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/teamengine#237]
If a test run (like in #236) fails cause of an exception in the test run the reponse contains only a very generic hint:
`Error executing test suite (wfs20-1.26)`   

Stacktrace from tomcat log:
```
SEVERE: Mapped exception to response: 500 (Internal Server Error)
javax.ws.rs.WebApplicationException
	at com.occamlab.te.spi.jaxrs.resources.TestRunResource.executeTestRun(TestRunResource.java:196)
	at com.occamlab.te.spi.jaxrs.resources.TestRunResource.handleGet(TestRunResource.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
 ```

and from the teamengine.log:
 ```
Sep 12, 2017 8:15:07 AM org.opengis.cite.iso19142.util.TestSuiteLogger log
WARNING: EPSG schema not found in DataSource--it will be created.
Sep 12, 2017 8:15:21 AM com.occamlab.te.spi.jaxrs.resources.TestRunResource executeTestRun
WARNING: Default CRS not recognized. No transform for classification “Krovak (North Orientated)”.
java.lang.RuntimeException: Default CRS not recognized. No transform for classification “Krovak (North Orientated)”.
	at org.opengis.cite.iso19142.FeatureTypeInfo.setSpatialExtent(FeatureTypeInfo.java:144)
	at org.opengis.cite.iso19142.util.ServiceMetadataUtils.extractFeatureTypeInfo(ServiceMetadataUtils.java:209)
	at org.opengis.cite.iso19142.SuiteFixtureListener.processWfsParameter(SuiteFixtureListener.java:95)
	at org.opengis.cite.iso19142.SuiteFixtureListener.onStart(SuiteFixtureListener.java:47)
	at org.testng.SuiteRunner.invokeListeners(SuiteRunner.java:208)
	at org.testng.SuiteRunner.run(SuiteRunner.java:266)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at com.occamlab.te.spi.executors.testng.TestNGExecutor.execute(TestNGExecutor.java:123)
	at org.opengis.cite.iso19142.TestNGController.doTestRun(TestNGController.java:141)
	at com.occamlab.te.spi.jaxrs.resources.TestRunResource.executeTestRun(TestRunResource.java:186)
	at com.occamlab.te.spi.jaxrs.resources.TestRunResource.handleGet(TestRunResource.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
 ```

Expected is that the detailed message is returned as in a test run with the web GUI.

**Comment by dstenger** [comment]
**Decision:**

The feedback given to the user should be the error message (if it is possible, the same message should appear as displayed by Web Browser Interface).
The stack trace itself must not be returned.

**Comment by keshav-nangare** [comment]
Fixed with this commit: 619cc9bbd91edecbbe7511dca3ed890d5e36252a

**Comment by dstenger** [comment]
## Test fix

* Start test environment with Docker (https://github.com/opengeospatial/teamengine-docker).
* Execute following request ("&" is not encoded): http://localhost:8081/teamengine/rest/suites/wfs20/1.26/run?wfs=http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS&request=GetCapabilities
  * HTTP status code of response is ```500```.
  * Error message is ```Error executing test suite (wfs20-1.26): Error message: Failed to parse resource located at http://cite.deegree.org/deegree-webservices-3.4-RC3/services/wfs200?service=WFS```
